real_time/arduino_src/radio_buggy_mega/main.cpp

```diff
@@ -65,6 +65,8 @@
 #define BRAKE_INDICATOR_PORT PORTE
 #define BRAKE_INDICATOR_PINN PE3 // arduino 5
 
+#define STATE_THRESHOLD 120 //This is a temporary change to eliminate magic number
```

Comment:
I was unsure if the two thresholds were related, so since they were the same value, I put in a temporary definition until someone complained that they were supposed to be treated differently